chore(feedback): Update info on self hosted flags and support (#10942)
getsentry/sentry-docs: develop-docs/application/feedback-architecture.mdx

---

## Self-hosted support

This feature is available if you enable the following feature flags:

- `organizations:user-feedback-ingest`
- `organizations:user-feedback-ui`
- `organizations:user-feedback-replay-clip`

Auto-registered flags for issue platform (you won't be able to grep these in the sentry repo, but you need to enable them too):

- `organizations:feedback-ingest`
- `organizations:feedback-visible`
- `organizations:feedback-post-process-group`

> ---

Reviewer: For this, you should also add that by version 24.8.0 (probably), user feedback is available on feature-complete version of sentry. Since we have "errors-only" and "feature-complete", so we won't encounter people mistakenly trying to enable user feedback while they're on errors-only version.

---

## Self-hosted support

This feature is available if you enable the following feature flags:

- `organizations:user-feedback-ingest`
- `organizations:user-feedback-ui`
- `organizations:user-feedback-replay-clip`

Auto-registered flags for issue platform (you won't be able to grep these in the sentry repo, but you need to enable them too):

- `organizations:feedback-ingest`
- `organizations:feedback-visible`
- `organizations:feedback-post-process-group`

> User feedback will be available on a feature-complete version of Sentry by v24.8.0.

Reviewer 2: The set up docs mention 24.4.2 for full functionality of UF, and 24.5.1 for attachments to be sent in the same envelope as feedback

Reviewer: Uhh.. Wouldn't this caused misunderstandings? Since I believe on early days, the required feature flags for enabling User Feedback is quite different than what's currently being set?

Also I don't think `organizations:user-feedback-replay-clip` is available on 24.4.2.

Author: I think it's fine @aldy505 , if it helps I'll add a note in the feature flag section that the flags apply for 24.7.1+.

---

## Self-hosted support

This feature is available if you enable the following feature flags.

- `organizations:user-feedback-ingest`
- `organizations:user-feedback-ui`
- `organizations:user-feedback-replay-clip`

Auto-registered flags for issue platform (you won't be able to grep these in the sentry repo, but you need to enable them too):

- `organizations:feedback-ingest`
- `organizations:feedback-visible`
- `organizations:feedback-post-process-group`

> You need to have v24.4.2 or higher in order to use the full functionality of UF
> (ex: screenshots). Lower versions may have limited functionality. Also note
> feedback is only available in feature-complete Sentry, _not_ errors-only.

Reviewer 2: ```suggestion
You need to have v24.4.2 or higher in order to use the full functionality of User Feedback
such as feedback with screenshots. Lower versions may have limited functionality. Also note
feedback is only available in feature-complete Sentry, _not_ errors-only.
```

---

> Auto/AI spam filters are currently unsupported. Make sure `organizations:user-feedback-spam-filter-ingest` is false.

Reviewer 2: I don't think we want to mention AI with our spam filtering
```suggestion
Auto spam filters are currently unsupported. Make sure `organizations:user-feedback-spam-filter-ingest` is false.
```
